nvflare/app_common/abstract/model.py

```diff
@@ -19,14 +19,22 @@
 """
 from nvflare.apis.dxo import DXO, DataKind
 
-from .learnable import Learnable as ModelLearnable
+from .learnable import Learnable
 
 
 class ModelLearnableKey(object):
     WEIGHTS = "weights"
     META = "meta"
 
 
+class ModelLearnable(Learnable):
+    def is_empty(self):
```

Comment:
Need to add is_empty() method to the Learnable class with a default of "return False".
ModelLearnable overrides this method here.

Reply:
Makes sense. Added.